gruntfile.js

```diff
@@ -537,4 +439,11 @@ module.exports = function (grunt) {
       }
     }
   });
+
+  grunt.registerTask('vendor', 'vendor:<min|reg>', function(min) {
```

Comment:
I think some documentation for this task would be great(mainly to specify how values will be mapped).